PatientRegistrationApp/src/components/SinglePatientPersonalInfo.vue

```diff
@@ -150,7 +150,7 @@ export default {
       suffix: "",
       date: "",
       gender: "",
-      patientPhoto: "",
+      patientPhoto: undefined,
```

Comment:
I'd argue that since you're implementing the photo feature, you should handle the photo appearing on this page. In this instance in which we don't have a photo, we def shouldn't have all this white space where the photo would be...

Can you handle that change please? That is unless there's a convincing argument for why this should be a separate card. 

![image](https://user-images.githubusercontent.com/67079030/100696254-c4fd3800-3358-11eb-916b-bb5c6bd5d101.png)

Reply:
I'm still seeing this issue in the single patient workflow. 

Furthermore, in the household workflow it looks like there are weird changes to the edges? 
![image](https://user-images.githubusercontent.com/67079030/101297965-49d0d180-37f1-11eb-8bab-33d169eea9f0.png)

Reply:
Not sure what the cause is of the curvy top of the image.